packages/clay-tooltip/demos/index.html

```diff
@@ -20,6 +20,10 @@
 		<h1>Tooltip</h1>
 	</div>
 
+	<button style="margin:50px;position:absolute;top:500px;left:50px;" title="">
+		<h1>Should not render ClayTooltip!</h1>
```

Comment:
hey @diegonvs you can remove the `h1` tag. Just not to get that big. 🙂

Reply:
I completely forgot this, sorry 🙏